portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,106 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css">
     <link rel="stylesheet" href="style.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+    <script src="node_modules/jquery/dist/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <section class="hero is-light is-large">
+        <div class="hero-head">
+    <nav class="navbar">
+      <div class="container">
+        <div class="navbar-brand">
+          <a class="navbar-item">
+            <h1>Iris Liu</h1>
+          </a>
+          <span class="navbar-burger burger" data-target="navbarMenuHeroB">
+            <span></span>
+            <span></span>
+            <span></span>
+          </span>
+        </div>
+        <div id="navbarMenuHeroB" class="navbar-menu">
+          <div class="navbar-end">
+            <a class="navbar-item is-active" href="index.html">
+              Home
+            </a>
+            <a class="navbar-item" href="me.html">
+              About Me
+            </a>
+            <a class="navbar-item" href="photo.html">
+              Photography
+            </a>
+            <span class="navbar-item">
+              <a class="button is-inverted navbar-icon-background" href="https://linkedin.com/in/jiayi-liu-1010" target="_blank">
+                <span class="icon has-text-dark is-large">
+                  <i class="fab fa-linkedin"></i>
+                </span>
+              </a>
+            </span>
+            <span class="navbar-item">
+              <a class="button is-inverted navbar-icon-background" href="https://github.com/irisliu77" target="_blank">
+                <span class="icon has-text-dark is-large">
+                  <i class="fab fa-github-square"></i>
+                </span>
+              </a>
+            </span>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </div>
+
+  <div class="hero-body"> 
```

Comment:
This hero-body class is responsible for adding a ton of whitespace between your navigation bar and your photo. I'm not sure you want that - the space feels a bit awkward.

In general I would audit all your CSS classes and ensure you understand what each is doing - I suspect some of them are unnecessary.

Reply:
i did that actually on purpose. I have change the padding to smaller value now.